plugins/link/lang/en.js

```diff
@@ -13,6 +13,7 @@ CKEDITOR.plugins.setLang( 'link', 'en', {
 		title: 'Anchor Properties',
 		name: 'Anchor Name',
 		errorName: 'Please type the anchor name',
+		errorWhitespace: 'Anchor name cannot contain whitespaces',
```

Comment:
```suggestion
		errorWhitespace: 'Anchor name cannot contain any whitespace',
```

Whitespace is mostly used as an uncountable noun.

Reply:
I've changed it to a space character, not a whitespace.